components/profile/time.rs

```diff
@@ -157,6 +157,145 @@ impl Formattable for ProfilerCategory {
             ProfilerCategory::IpcReceiver => "Blocked at IPC Receive",
             ProfilerCategory::IpcBytesReceiver => "Blocked at IPC Bytes Receive",
             ProfilerCategory::ApplicationHeartbeat => "Application Heartbeat",
+            ProfilerCategory::WebGlGetContextAttributes => "WebGlGetContextAttributes",
```

Comment:
We could just do a `Derive(Debug)` here, but we would lose the `+ ` and `| +` formatting for subcategories. Although we could just derive debug, and then just manually add the prefix to the few categories that need them. The output would be like `IpcReceiver` rather than `Blocked at IPC Receive` though.